decidim-forms/README.md

```diff
@@ -2,6 +2,34 @@
 
 This gem encapsulates the logic to create and manage forms, so it can be reused in other modules, like surveys and meetings.
 
+A `Decidim::Forms::Question` must be of one of the types:
+
+- short_answer
+- long_answer
+- single_option
+- multiple_option
+- sorting
+
+Here are the relations between the classes of a `Decidim::Questionnaire`:
+
+```
+                  1..* +----------+         1..* +--------------+
+        +------------->| Question |------------->| AnswerOption |
+        |              +-----+----+              +------+-------+
+        |                    |                          ^ 1..*
```

Comment:
```suggestion
        |                    ^ 1..*                     ^ 1..*
```

Reply:
Thanks @rbngzlv for the note :smile: 

I've added it, but just with a 1..1 relation :+1: